core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -834,38 +807,24 @@ oppia.directive('conversationSkin', [
                     }
                     FatigueDetectionService.reset();
                     NumberAttemptsService.reset();
-                    _nextFocusLabel = FocusManagerService.generateFocusLabel();
-
-                    PlayerTranscriptService.setDestination(newStateName);
+                    _nextFocusLabel = focusLabel;
```

Comment:
Where is _nextFocusLabel (and therefore focusLabel) getting used? I don't see it being used anywhere...

Do we currently focus correctly when switching cards? I can't find the call that does that.

Reply:
It was used here, but then was shifted to after next card is added. Will remove this.

Reply:
Done.